### PR TITLE
Cherry-pick #16616 to 7.x: Vendor Agent dependencies

### PR DESCRIPTION
**Pull request by ph**
Cherry-pick of PR #16616 to 7.x branch. Original message: 

This PR is only the dependencies required by the Agent. We do this to simplify the Agent PR itself and to facilitate future backport to 7.x

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -4574,6 +4596,48 @@
...
+		},
+		{
+			"checksumSHA1": "QXKrSi4ZkCnYbKIhsWuxG/piSG0=",
+			"path": "github.com/urso/diag/ctxfmt",
+			"revision": "21b3cc8eb79749b97f4be4fa033ed8e82c007c80",
+			"revisionTime": "2020-02-10T12:31:36Z"
+		},
+		{
+			"checksumSHA1": "q+rF16PGsfdxqnP3h4xt03Rgl98=",
+			"path": "github.com/urso/ecslog",
```
Comment (ruflin) [inline vendor/vendor.json]: @urso Should we bring this into a elastic repo long term?

Comment (urso) [reply vendor/vendor.json]: Long term and if there is still interest in ecslog, yes.

**Comment by urso** [comment]
@ph looks like switch to go modules to add a conflict.

**Comment by ph** [comment]
yeah, didn't have conflict yesterday rebasing it.

**Comment by ph** [comment]
rebased waiting for ci green to merge

**Comment by ph** [comment]
going to close this, with the introduction of go module it will break this PR.

**Pull request closed without merging**